libs/ngx-dsfr/password/src/lib/password.component.html

```diff
@@ -34,13 +34,16 @@
       minimum
     </p>
   </div>
-  <div class="fr-password__checkbox">
-    <dsfr-button
-      [label]="buttonDisplayPasswordText"
-      [icon]="buttonDisplayPasswordIcon"
-      [type]="buttonType.TERTIARY_NO_OUTLINE"
-      (click)="displayPassword()">
-    </dsfr-button>
+  <div class="fr-password__checkbox fr-checkbox-group fr-checkbox-group--sm">
+    <input
+      type="checkbox"
+      aria-label="Afficher le mot de passe"
+      [id]="id + '-show'"
+      [attr.aria-describedby]="id + '-show-messages'"
+      (change)="displayPassword()">
+    <label class="fr-password__checkbox fr-label"  [for]="id + '-show'">
+      Afficher
+    </label>
   </div>
```

Comment:
You forgot to add this. I didn't what it was but apparently it's important for accessibility [reasons](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) : 
```html
<div
      class="fr-messages-group"
      [id]="id + '-show-messages'"
      aria-live="assertive">
      {{ isPasswordDisplayed ? "Le mot de passe est affiché" : "" }}
    </div>
```

Obviously, it's for screen-readers only so we need to change the CSS too : 
```css
// Imports
@use "@gouvfr/dsfr/dist/component/form/form.min";
@use "@gouvfr/dsfr/dist/component/input/input.min";
@use "@gouvfr/dsfr/dist/component/checkbox/checkbox.min";
@use "@gouvfr/dsfr/dist/component/password/password.min";

// Actual Styling
.fr-password__checkbox .fr-messages-group {
  width: 0;
  height: 0;
  overflow: hidden;
}
```

And finally, you're going to need to add these changes to the `ts` file : 

- No need for `import { ButtonType }` on line 16
- You can remove `buttonType` on line 89
- You need to add this `isPasswordDisplayed: boolean = false;` and handle it in `displayPassword`